site/search/SearchPanel.tsx

```diff
@@ -510,17 +306,6 @@ const SearchResults = (props: SearchResultsProps) => {
                         "data-algolia-position"
```

Comment:
`data-algolia-event-name` and `eventName`, a few lines above, is now also unused.

Reply:
I removed `eventName`, but what do you mean by `data-algolia-event-name` is now unused? Is there another snippet in this file I'm blind to that I need to remove?

Reply:
No all good now, sorry for being unclear about it.